common/infrastructure/src/main/kotlin/no/nav/su/se/bakover/common/infrastructure/persistence/Session.kt

```diff
@@ -119,6 +120,7 @@ open class Session(
                 is java.sql.Array -> this.setArray(idx, v)
                 is URL -> this.setURL(idx, v)
                 is UUID30 -> this.setString(idx, v.toString())
+                is BehandlingsId -> this.setString(idx, v.value.toString())
```

Comment:
Kan vi legge på en also og log.warn her? For å potensielt kunne fjerne den herfra igjen :shrug: (du nevnte at du ikke ønsket den her egentlig)